pmm-app/tests/ia/pages/ruleTemplatesPage.js

```diff
@@ -0,0 +1,86 @@
+const { I } = inject();
+const fs = require('fs');
+const YAML = require('yaml');
+
+module.exports = {
+  url: 'graph/integrated-alerting',
+  columnHeaders: ['Name', 'Source', 'Created', 'Actions'],
+  elements: {
+    addedTemplate: '//td[text()="TemplateForAutomation"]/following-sibling::td[text()="User-defined (UI)"]',
+    ruleTemplateTab: '//li[@aria-label="Tab Alert Rule Templates"]',
+    templatesTableHeader: '$alert-rule-templates-table-thead',
+    templatesTable: '$alert-rule-templates-table-tbody',
+    templateName: '//tr/td[1]',
+    modalHeader: '$modal-header',
+    popUpTitle: '.alert-title',
+    columnHeaderLocator: columnHeaderText => `//th[text()="${columnHeaderText}"]`,
+  },
+  buttons: {
```

Comment:
buttons & fields, could be part of page elements? 

I don't have a strong opinion on this, just a suggestion.

Reply:
I can move them to the `elements` object. My intention was to have different objects: `elements` - non-interectable (static selectors), `buttons` - buttons, and related methods, `fields` - fields, `messages` - static messages and related methods. So should I move them to the `elements`?

Reply:
okay makes sense, no need to move.